dokku

```diff
@@ -2,6 +2,8 @@
 set -eo pipefail
 shopt -s nullglob
 
+[[ $DOKKU_TRACE ]] && set -x
+
```

Comment:
amending, but the answer was because non dokku user's call the dokku command again with sudo :wink: